deploy/rp-development.json

```diff
@@ -116,14 +134,52 @@
                     }
                 ]
             },
-            "name": "rp-vnet",
+            "name": "pe-static-vnet",
```

Comment:
Is this name final?

Reply:
rolled back to the one specified in ticket.